AutoWorkup/workflows/WorkupJointFusion.py

```diff
@@ -151,6 +151,7 @@ def CreateJointFusionWorkflow(WFname, onlyT1, master_config, runFixFusionLabelMa
     :: Resampling is placed at this stage
     """
     subjectT2Resample = pe.Node(interface=BRAINSResample(), name="BRAINSResample_T2_forAntsJointFusion")
+     HACK: NEED TO FIX onlyT1 that was removed from other location.
```

Comment:
Oh.. I did work on and made changes to make Joint Fusion t1 only while keeping registration multimodal. 
Sorry, I was waiting for testing to finish. 

I will review the code and merge those changes properly today.

Are you working on the code???

Regina
> On May 26, 2016, at 10:13 PM, Hans Johnson <notifications@github.com> wrote:
> 
> In AutoWorkup/workflows/WorkupJointFusion.py <https://github.com/BRAINSia/BRAINSTools/pull/276#discussion_r64852245>:
> 
> > @@ -151,6 +151,7 @@ def CreateJointFusionWorkflow(WFname, onlyT1, master_config, runFixFusionLabelMa
> >      :: Resampling is placed at this stage
> >      """
> >      subjectT2Resample = pe.Node(interface=BRAINSResample(), name="BRAINSResample_T2_forAntsJointFusion")
> > +     HACK: NEED TO FIX onlyT1 that was removed from other location.
>  <https://camo.githubusercontent.com/37ddd55708a38212f4e9de4e3eac98358fefe597/68747470733a2f2f7777772e636f646163792e636f6d2f6173736574732f696d616765732f66617669636f6e2e706e67> Issue found: unexpected indent <https://www.codacy.com/app/hans-j-johnson/BRAINSTools/file/2643618562/issues/source?bid=3194574&fileBranchId=3322153#l154>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/BRAINSia/BRAINSTools/pull/276/files/940cee5371be33542a3634e9f3094d4840d85273#r64852245>, or mute the thread <https://github.com/notifications/unsubscribe/AA82ocbLjOyS-mvRQlPypRcS5GQaxiSKks5qFmFegaJpZM4IoIqO>.
>